docs/pct/README.md

```diff
@@ -0,0 +1,291 @@
+# Puppet Content Templates
+
+* [Overview](#overview)
+* [Getting Started](#getting-started)
+  * [pct new](#pct-new)
+  * [Template Updates](#template-updates)
+  * [More Templates](#more-templates)
+* [Writing Your Own Templates](#writing-templates)
+  * [Dos and Don'ts](#dos-and-donts)
+  * [Structure](#structure)
+    * [pct-config.yml](#pct-configyml)
+  * [Templating Language](#templating-language)
+  * [Testing](#testing)
+  * [Sharing](#sharing)
+* [Overriding Template Defaults](#overriding-template-defaults)
+
+## Overview
+
+`Puppet Content Templates` (PCT) allow you to quickly create Puppet content from scaffolding templates built by Puppet and the Community.
+
+> :warning: PCT is currently in an EXPERIMENTAL phase and feedback is encouraged via [pdk/discussions](https://github.com/puppetlabs/pdk/discussions) and starting a `feedback` post.
```

Comment:
Should we make a discussions tab in the pdkgo repo instead of mixing in the pdk repo?

Reply:
Sounds good